website/docs/enterprise/installation/airgap.mdx

```diff
@@ -0,0 +1,560 @@
+---
+title: Airgap environments
+sidebar_position: 1
+hide_title: true
+toc_max_heading_level: 4
+---
+
+# Installing Weave Gitops Enterprise in Airgap environments
+
+From [wikipedia](https://en.wikipedia.org/wiki/Air_gap_(networking))
+
+>An air gap, air wall, air gapping or disconnected network is a network security measure employed on one or more computers
+to ensure that a secure computer network is physically isolated from unsecured networks, such as the public Internet or an unsecured local area network...
+
+This document guides on how to install Weave Gitops Enterprise in a restricted environment.
+
+# Restrictions
+
+There are many restriction could happen within an airgap environment. This guide assumes that you have egress network
+restrictions. In order to install Weave Gitops Enterprise (WGE), the required artifacts are required to be loaded
+from a private registry. This guide helps you with the task to identity the helm charts
+and container images required to install WGE and to load them into your private registry.
+
+A second assumption is that you could prepare the installation from a proxy host that has internet
+access to push artifacts into your private registry. We will refer it as proxy or proxy host. 
+
+# Install
+
+There are different variations of the following stages and conditions. We consider that installing 
+Weave Gitops Enterprise in an airgap environment could follow the following stages.
+
+1. Setup a WGE install environment.
+2. Collect artifacts and publish to a private registry.
+3. Install Weave Gitops Enterprise in the airgap environment.
+
+## Setup a WGE install environment
+
+The main goal of this stage is to recreate a local Weave Gitops Enterprise within your context, to collect
+the container images and helm charts, that will be required in your private registry for the offline installation.
+
+A three-step setup is followed.
+
+1. Setup a proxy host
+2. Setup a private registry
+3. Install Weave Gitops Enterprise
+
+### Setup a proxy host
```

Comment:
I think a proxy host assumption for some customers like banks will be fine, would be good to check with our most secure customers if they can also use a proxy host.

Reply:
I checked this and we would have to be compatible with a laptop to download the artefacts then disconnect from the internet and connect to the secure network and then be able to upload the artefacts to the private registry.
Therefore an intermediate step of writing out a tarball and then uploading in a separate step.

Reply:
thanks for checking, yes proxy host here should be understood as a host that have access to both networks, could be of multiple forms. adding that.

Reply:
updated